zerver/lib/webhooks/common.py

```diff
@@ -96,6 +97,24 @@ def check_send_webhook_message(
             # webhook-errors.log
             pass
 
+def parse_headers_dict(custom_headers: Union[None, Dict[str, Any]]) -> Union[None, Dict[str, str]]:
+    """ A modified version of parse_headers from
+    zerver/management/commands/send_webhook_fixture_message. """
+    headers = {}
+    if not custom_headers:
+        return {}
+    for header in custom_headers:
+        if len(header.split(" ")) > 1:
+            raise ValueError("custom header '%s' contains a space." % (header,))
```

Comment:
Can a custom header contain a space?

Reply:
From my experience, I've never encountered a HTTP Header where spaces were used (whether it was a standard header of a custom header). So for the time being, it think that it wouldn't be a bad idea to allow spaces.